src/partition/FileUtils.h

```diff
@@ -59,6 +64,36 @@ class InputFile {
     off_t _sz;
 };
 
+class InputFileArrow : public InputFile {
+public:
+    explicit InputFileArrow(boost::filesystem::path const &path, off_t blockSize);
+    ~InputFileArrow();
+
+    /// Return the size of the input file.
+    off_t size() const { return _sz; }
+
+    /// Return the path of the input file.
+    boost::filesystem::path const &path() const { return _path; }
```

Comment:
The code formatting is incorrect., You have extra space in `const &path(` and no space before the method name. Should be `const& path(`.  Please, run this tool on your code before each commit:
```
qserv build -j24 --no-cmake --no-make --no-build-image --clang-format=REFORMAT
```

Reply:
Done